src/wasm-binary.h

```diff
@@ -1119,8 +1121,8 @@ class WasmBinaryBuilder {
         e->value = name;
```

Comment:
alternatively, we could create the export later down, after we have called `addImport/addFunction`, which would have set a numeric name.